github-release.sh

```diff
@@ -0,0 +1,55 @@
+SHA=$1
+PERSONAL_ACCESS_TOKEN=$2
+
+SHORT_SHA=$(echo ${SHA} | head -c 7)
+DATE=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
+
+echo "Creating Tag"
+tag_output=$(curl \
+    --request POST \
+    --header "Authorization: token $PERSONAL_ACCESS_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data "{
+        \"tag\": \"${SHA}\",
+        \"message\": \"automated tag for ${SHORT_SHA}\",
+        \"object\": \"${SHA}\",
+        \"type\": \"commit\",
+        \"tagger\": {
+            \"name\": \"taylorbot\",
+            \"email\": \"dev@giantswarm.io\",
+            \"date\": \"${DATE}\"
+        }
+    }" \
+    https://api.github.com/repos/giantswarm/e2e-harness/git/tags
```

Comment:
Could we parameterize the project name too? This way we could copy this file between projects as needed without changes.

Reply:
on it 👍